### ISSUE TITLE
DOI URL is read out in its entirety

### ISSUE THREAD

**Issue by benaltair** [bahaistudies/jbs#53]
The `<a>` link has the full URL as its text. To make a better screen reader experience, it needs `aria-label` added. The example below shows how this could work. This should be added to all locations that include a DOI link.
```html
<a href="https://doi.org/10.31581/jbs-29.4.2(2019)" aria-label="Document Object Identifier (D O I)">https://doi.org/10.31581/jbs-29.4.2(2019)</a>
```
